extensions/wikia/Wall/WallMessage.class.php

```diff
@@ -691,18 +569,18 @@ public function getWallPageUrl() {
 	}
 
 	/**
-	 * @param bool $useMasterDB
 	 * @return Title
 	 */
 	public function getArticleTitle( $useMasterDB = false ) {
-		$commentsIndex = $this->getCommentsIndex();
-
+		$commentsIndex = $this->getCommentsIndexEntry();
 		if ( empty( $commentsIndex ) ) {
+			$this->error( __METHOD__ . ' - SUS-1680 - No comments_index entry for message', [
+				'messageTitle' => $this->getTitle()->getPrefixedText(),
+				'messageId' => $this->getTitle()->getArticleID()
```

Comment:
![MINOR](https://raw.githubusercontent.com/SonarCommunity/sonar-github/master/images/severity-minor.png) Define a constant instead of duplicating this literal "messageId" 5 times. [![rule](https://raw.githubusercontent.com/SonarCommunity/sonar-github/master/images/rule.png)](https://sonarqube.wikia-inc.com/coding_rules#rule_key=php%3AS1192)